src/Add.c

```diff
@@ -34,7 +36,7 @@ void add() {
         }
     }
 
-    
+
     system("cls"); /* clear screen */
     fseek(fp, 0, SEEK_END);

```

Comment:
Handle the error if it returns -1

Reply:
This is been handled by our file_open function. It test the the file open open first after that marked logic is executed.